frb_codegen/src/ir/ty_struct.rs

```diff
@@ -56,6 +57,7 @@ pub struct IrStruct {
     pub is_fields_named: bool,
     pub dart_metadata: Vec<IrDartAnnotation>,
     pub comments: Vec<IrComment>,
+    pub is_empty: bool,
```

Comment:
Hmm we have `fields`, so is it possible that we can derive is_empty from `self.fields.is_empty()`, and thus remove this new field